pages/migration/wsc-53/migration_wsc-53_php.md

```diff
@@ -75,3 +75,13 @@ The reCAPTCHA v1 endpoints have already been turned off by Google and always ret
 Thus the implementation was completely non-functional even before this change.
 
 See [WoltLab/WCF#3781](https://github.com/WoltLab/WCF/pull/3781) for details.
+
+## Search
+
+The generic implementation in the `AbstractSearchEngine::parseSearchQuery()` method was dangerous, because it did not have knowledge about the search engine’s specifics.
+The implementation was completely removed `parseSearchQuery()` now always throws a `\BadMethodCallException`.
+
+If you implemented a custom search engine and relied on this method you can inline the previous implementation to preserve existing behavior.
+You should take the time to verify the results against the manual of the search engine to make sure it cannot generate malformed queries or security issues.
```

Comment:
"results" is too vague here and should be replaced with a more specific word/phrase.

Reply:
How about this:

```suggestion
You should take the time to verify the generated queries against the manual of the search engine to make sure it cannot generate malformed queries or security issues.
```

Reply:
Did s/generated/rewritten/ in my suggestion.

Reply:
Yes, that is better!